update resources.md (#2133)
SumoLogic/sumologic-documentation: docs/observability/aws/deploy-use-aws-observability/resources.md

   </td>
  </tr>
  <tr>

>    <td>Lambda Log Forwarder ( AWS CloudWatch logs)

Reviewer: ```suggestion
   <td>Lambda Log Forwarder (AWS CloudWatch logs)
```

## Resources created in Sumo Logic

> ### Meta-data Tags

Reviewer: no dash please

```suggestion
### Metadata Tags
```

> The Meta-data tags applied on Sumo Logic Sources.

Reviewer: ```suggestion
The metadata tags applied on Sumo Logic Sources.
```
